packages/botonic-nlp/tests/tasks/text-classification/botonic-text-classifier.test.ts

```diff
@@ -30,6 +30,29 @@ describe('Botonic Text Classifier', () => {
     expect(classifier.vocabulary.length).toEqual(8)
   })
 
+  test('Train and Evaluate Model', async () => {
+    const dataset = classifier.loadDataset(generalHelper.DATA_DIR_PATH)
+    const { trainSet, testSet } = classifier.splitDataset(dataset, 0.5, false)
+    classifier.generateVocabulary(trainSet)
+    try {
+      classifier.compile()
+    } catch (e) {
+      console.log(e)
```

Comment:
why? remove catch if possible

Reply:
The catch is a way to avoid that the compile method throws an Error because the PredictionProcessor is not implemented. Maybe it is better to remove this test until the PredictionProcessor is implemented and no errors are thrown.